dockerfiles/base/scripts/base/startup.sh

```diff
@@ -251,20 +255,40 @@ init() {
   source "${SCRIPTS_CONTAINER_SOURCE_DIR}"/cli.sh
 }
 
+cleanup() {
+  RETURN_CODE=$?
+
+  # CLI developers should only return '3' in code after the init() method has completed.
+  # This will check to see if the CLI directory is not mounted and only offer the error
+  # message if it isn't currently mounted.
+  if [ $RETURN_CODE -eq "3" ]; then
```

Comment:
In the current CLI implementation, there are no methods that return a '3'. So, this is laying a groundwork so that as we improve the CLI over time, if there are CLI crashes or hard error messages that we print out, we should return this error code to trigger this help message.